opentracing-spring-cloud-starter-jaeger/src/main/java/io/opentracing/contrib/spring/cloud/starter/jaeger/JaegerConfigurationProperties.java

```diff
@@ -0,0 +1,302 @@
+  /**
+   * If this options is enabled, then the instantiation of the Tracer will be handed off to the
+   * {@link com.uber.jaeger.tracerresolver.JaegerTracerResolver} Enabling this option means that
+   * none of the autoconfigured beans apply. Furthermore the mere setting of this option to true is
+   * not enough for it to take effect, {@link com.uber.jaeger.tracerresolver.JaegerTracerResolver}
+   * also needs to be on the classpath
+   */
+  private boolean useTracerResolver = false;
+  /**
+   * The serviceName that the tracer will use
+   */
+  private String serviceName = "spring-boot";
+  /**
+   * Whether spans should be logged to the console
+   */
+  private boolean logSpans = false;
+  /**
+   * Enable the handling of B3 headers like "X-B3-TraceId" This setting should be used when it is
+   * desired for Jaeger to be able to join traces started by other Zipkin instrumented applications
+   */
+  private boolean enableB3Propagation = false;
+  /**
+   * Enable the collections of metrics of collection process Setting this to true will enable {@link
+   * com.uber.jaeger.metrics.InMemoryStatsReporter}
+   */
+  private boolean enableMetrics = false;
+
+  public boolean isEnabled() {
+    return enabled;
+  }
+
+  public void setEnabled(boolean enabled) {
+    this.enabled = enabled;
+  }
+
+  public boolean isUseTracerResolver() {
+    return useTracerResolver;
+  }
+
+  public void setUseTracerResolver(boolean useTracerResolver) {
+    this.useTracerResolver = useTracerResolver;
+  }
+
+  public String getServiceName() {
+    return serviceName;
+  }
+
+  public void setServiceName(String serviceName) {
+    this.serviceName = serviceName;
+  }
+
+  public boolean isLogSpans() {
+    return logSpans;
+  }
+
+  public void setLogSpans(boolean logSpans) {
+    this.logSpans = logSpans;
+  }
+
+  public boolean isEnableB3Propagation() {
+    return enableB3Propagation;
+  }
+
+  public void setEnableB3Propagation(boolean enableB3Propagation) {
+    this.enableB3Propagation = enableB3Propagation;
+  }
+
+  public boolean isEnableMetrics() {
+    return enableMetrics;
+  }
+
+  public void setEnableMetrics(boolean enableMetrics) {
+    this.enableMetrics = enableMetrics;
+  }
+
+  public HttpSender getHttpSender() {
+    return httpSender;
+  }
+
+  public RemoteReporterProperties getRemoteReporterProperties() {
+    return remoteReporterProperties;
+  }
+
+  public UdpSender getUdpSender() {
+    return udpSender;
+  }
+
+  public ConstSampler getConstSampler() {
+    return constSampler;
+  }
+
+  public ProbabilisticSampler getProbabilisticSampler() {
+    return probabilisticSampler;
+  }
+
+  public RateLimitingSampler getRateLimitingSampler() {
+    return rateLimitingSampler;
+  }
+
+  public RemoteControlledSampler getRemoteControlledSampler() {
+    return remoteControlledSampler;
+  }
+
+
+  public static class RemoteReporterProperties {
+
+    private int flushInterval = 1000;
+
+    private int maxQueueSize = 100;
```

Comment:
Jaeger client 0.26 has a better builder API. It shouldn't require to specify these "default" values.

Reply:
I'm looking into utilizing the new API

Reply:
Updated using the builders